net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -995,6 +996,11 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
     kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
 );
 
+// Will be initialized to a sorted list of headers not allowed in
+// AddResourceHeader.
+const StringPieceVector* fixed_resource_headers_ = NULL;
+bool http_header_separators_[256] = {false};
```

Comment:
Remove the trailing underscore -- those are for member vars and do not connote private or file-local.

I am not aware of any way to distinguish namespaced constants from locals via naming convention.